guides/common/modules/proc_deploying-a-custom-ssl-certificate-to-smart-proxy-server.adoc

```diff
@@ -14,48 +14,24 @@ For more information, see {InstallingSmartProxyDocURL}registering-to-satellite-s
 For more information, see {InstallingSmartProxyDocURL}installing-capsule-server-packages_{smart-proxy-context}[Installing {SmartProxyServer} Packages].
 
 .Procedure
-. On your {ProjectServer}, validate the custom SSL certificate input files:
+. On your {ProjectServer}, generate a certificate bundle:
 +
+.Example output of `katello-certs-check`
```

Comment:
```suggestion
Example output of `katello-certs-check`:
```
If below is the output of katello-certs-check, then the step with running it should not have been omitted.